cmd/ddev/cmd/global_dotddev_assets/commands/host/tableplus

```diff
@@ -7,10 +7,9 @@
 ## Usage: tableplus
 ## Example: "ddev tableplus"
 ## OSTypes: darwin
-## HostBinaryExists: /Applications/TablePlus.app
+## HostBinaryExists: /Applications/TablePlus.app,/Applications/Setapp/TablePlus.app
```

Comment:
I don't understand the usage of `/Applications/Setapp/TablePlus.app` - Is /Applications/Setapp a directory that is commonly used for installs? I've never seen it. 

If this is just something for your personal use, then just remove the #ddev-generated line at the beginning of the script and manage it yourself.

Reply:
Oh, also, I think you must have done the `make packr2` on Windows? I haven't figured out how or why, but it does fail to do that correctly on Windows. We just need to make sure that it gets done on macos or linux before merge.

Reply:
If you have the Setapp installed, it installs applications to it's own directory instead of the default applications directory, as it's a subscription service where you get a bunch of paid software rather cheaply it has become popular with mac users in our office.

I've made all pull requests and modifications on my mac, as I'm with family and don't have access to my windows desktop,  should it have returned any specific output? I've just run it again and the output is still just the normal docker run command output.

Reply:
Your Windows desktop won't be able to demonstrate anything anyway, unfortunately, only the mac can do it. Windows TablePlus doesn't have the ability to be launched with a dburl, https://github.com/TablePlus/TablePlus/issues/2224

To test this, rebase it and run `make packr2` and commit the results if anything changes, then `make` and run the resultant binary. 

I'm reluctant to make a change though that seems very special to your environment, when you can just edit the file in your own ~/.ddev/commands/host and be done with it.

Reply:
I think that you can just close this commit, I agree that since it can be done locally and is a more niche environment, it shouldn't have to be supported out of the box, and we already run with the customized version just fine.